_includes/footer.html

```diff
@@ -22,6 +22,18 @@
         <span class="username">{{ site.github_username }}</span>
       </a>
     </li>
+       <li>
+      <a href="https://mastodon.social/@opensourcedesign{{ site.mastodon_username }}">
```

Comment:
this should probably be `https://mastodon.social/{{ site.mastodon_username }}`, and the site needs to be configured to have a mastodon_username.

Reply:
hmmm i'm not sure how to put that in the yaml but i'll look into it!